python/social_bot/envs/grocery_ground.py

```diff
@@ -195,20 +223,20 @@ def __init__(self,
             obs_data_space = gym.spaces.Box(
                 low=-50, high=50, shape=obs_data.shape, dtype=np.float32)
 
-        control_space = gym.spaces.Box(
+        self._control_space = gym.spaces.Box(
             low=-1.0,
             high=1.0,
             shape=[len(self._agent_joints)],
             dtype=np.float32)
 
         if self._with_language:
             self.observation_space = gym.spaces.Dict(
-                data=obs_data_space, sentence=DiscreteSequence(128, 24))
+                data=obs_data_space, sentence=DiscreteSequence(256, 24))
             self.action_space = gym.spaces.Dict(
-                control=control_space, sentence=DiscreteSequence(128, 24))
+                control=self._control_space, sentence=DiscreteSequence(256, 24))
```

Comment:
We still need to convert this to integer sequence somewhere. If it's easy to do on alf side (i.e. tensorflow has related ops), we can do it on alf side. Otherwise, we need to convert sentence to integer sequence.

Reply:
I added a helper function to class DiscreteSequence, to convert a sentence to integer sequence by subword segmentation based on BPEmb.